Makefile

```diff
@@ -1,6 +1,7 @@
 SHELL := /bin/bash
 PKG = github.com/Clever/csvlint
-PKGS = $(PKG)
+SUBPKGS = $(addprefix $(PKG)/, cmd/csvlint)
+PKGS = $(PKG) $(SUBPKGS)
```

Comment:
`PKG`, `SUBPKGS`, and `PKGS` should probably be `:=` assignments: http://stackoverflow.com/a/448939/205138